_posts/2023-08-31-governance-v1.md

```
@@ -0,0 +1,552 @@
+The goal of this democratic governance model is to allow stakeholders to have a say in the direction and evolution of the network, and to ensure that the network is run in a decentralized and transparent manner. By allowing stakeholders to participate in the decision-making process, Polkadot aims to create a more decentralized, transparent, and fair governance system for its blockchain network.
+
+The section is primarily divided into two parts →
+
+1. **Democracy Proposals**
+2. **Referendum**
+
+## Democracy Proposals
+
+### Creating a Democracy Proposal
+
+**Process**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)